ONSdigital/eq-author-app
# Removed isNew condition for displaying validation errors

### What is the context of this PR?
To display validation errors on creation of page or answer

### How to review 
1. Create a page
2. Validation error should immediately show for title
3. Create Text or Multiple choice answer
4. Validation error message should show immediately
